types/subleveldown/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for subleveldown 4.1
+// Project: https://github.com/level/subleveldown
+// Definitions by: Carson Farmer <https://github.com/carsonfarmer>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { LevelUp } from 'levelup';
+import { AbstractLevelDOWN, AbstractIterator, ErrorCallback } from 'abstract-leveldown';
+
+declare namespace sub {
```

Comment:
Is this the correct use of a namespace here?

Reply:
As in, should this be named `SubDown`?

Reply:
I think not.  In the [source code](https://github.com/Level/subleveldown/blob/master/index.js#L5), a function is exported, not a class. The class instance only returns from the function.

I probably made some unintentional mistakes in some comments because I got confused in the source code of the library.

I can review this PR in more detail and thoughtfully. What do you say about this?

Reply:
Yes please, and not a problem at all!

Reply:
I suggest such improvements: carsonfarmer/DefinitelyTyped#1.